### PR TITLE
To Add Appointment models and views , Medical History models and views , updated the mess_type field to mess_meal field and fixed some false imports

### PR DESCRIPTION
**Pull request by architprasar**
@ashutoshc8101 please review the pr 
Changes made:
- Added appointment and medical history models, serializers, views, and URLs.
- Fixed the false import of module named curses at api/mess/serializers.py
- fixed the name of field mess_type to mess meal as it was declared twice

### CODE REVIEW

File: api/medical/views.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 from rest_framework.views import APIView
 from rest_framework.permissions import IsAuthenticated, AllowAny
 from rest_framework.response import Response
-
+from authentication.models import User
 
 # views for medical manager
+
+
 class AddDoctorView(APIView):
```
Comment (ashutoshc8101) [inline api/medical/views.py:12]: Remove these two empty lines.

File: api/medical/views.py [hunk 2]
```diff
@@ -170,3 +172,78 @@ def get(self, request, *args, **kwargs):
...
+        doctor = request.POST.get("doctor", None)
+        doctorinstance = Doctor.objects.filter(id=doctor).first()
+        user = request.user
+        instance = Appointment.objects.create(
+            doctor=doctorinstance,
+            patient=user
+        )
+        instance.save()
+        serinstance = AppointmentSerializer(instance)
+        return Response(serinstance.data)
```
Comment (ashutoshc8101) [inline api/medical/views.py:190]:
```suggestion
        return Response(AppointmentSerializer(instance).data)
```

File: api/medical/views.py [hunk 3]
```diff
@@ -170,3 +172,78 @@ def get(self, request, *args, **kwargs):
...
+            time = request.POST.get("time", None)
+            appointment.date = date
+            appointment.time = time
+            appointment.accepted = True
+        else:
+            reason = request.POST.get("reason", None)
+            appointment.reason = reason
+            appointment.accepted = False
+        appointment.save()
+        serinstance = AppointmentSerializer(appointment)
```
Comment (ashutoshc8101) [inline api/medical/views.py:223]: ditto everywhere

File: api/mess/models.py [hunk 4]
```diff
@@ -40,18 +40,13 @@ class MessMenu(models.Model):
...
-    weekdays = models.ManyToManyField(WeekDay)
-
 # data base model for feedback
 # status is for whether the feedback is viewed by mess manager or not.
 class MessFeedback(models.Model):
     user = models.ForeignKey(settings.AUTH_USER_MODEL,
                              on_delete=models.CASCADE)
     body = models.TextField(editable=True, null=False, blank=True)
-    mess_type = models.CharField(max_length=50)
+    mess_meal = models.CharField(max_length=50)
```
Comment (ashutoshc8101) [inline api/mess/models.py:49]: Why is this required? Have you made sure that this won't break other things?

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]

**Comment by ashutoshc8101** [comment]
@architprasar Left a few comments. Also, provide a proper PR name briefly describing what the PR does.

File: api/mess/models.py (hunk 4 above)
Comment (architprasar) [reply api/mess/models.py:49]: yes this is necessary as per the feedback designed for the mess

File: api/medical/views.py (hunk 3 above)
Comment (architprasar) [reply api/medical/views.py:222]: unable to understand the issue can you please breif

**Review by architprasar — COMMENTED** [review]
changes made as asked

File: api/medical/views.py (hunk 1 above)
Comment (architprasar) [reply api/medical/views.py]: done

**Comment by architprasar** [comment]
@ashutoshc8101 please review again

File: api/mess/models.py (hunk 4 above)
Comment (ashutoshc8101) [reply api/mess/models.py:49]: Won't the frontend model also change? Changing this only in the backend will cause errors.

**Comment by ashutoshc8101** [comment]
@architprasar left a comment

File: api/mess/models.py (hunk 4 above)
Comment (architprasar) [reply api/mess/models.py:49]: no changes are required at the front end as payload names for the API are the same only the model field is changed. if any changes are required then are requested to be made.

**Review by ashutoshc8101 — APPROVED** [review]
